components/event-publisher-proxy/pkg/receiver/receiver.go

```diff
@@ -22,21 +22,22 @@ const (
 
 // HTTPMessageReceiver is responsible for receiving messages over HTTP.
 type HTTPMessageReceiver struct {
-	port     int
+	Host     string
```

Comment:
Thats exactly where we need this field. If we set it to something other than localhost during tests on a mac, then it will cause a popup from macos asking for firewall permissions